test/IdRegistry/IdRegistry.t.sol

```diff
@@ -300,6 +300,74 @@ contract IdRegistryTest is IdRegistryTestSuite {
         assertEq(idRegistry.getRecoveryOf(1), address(0));
     }
 
+    function testFuzzVerifyFidSignature(
```

Comment:
nit add comment headers to separate and make it easier to read, like: 

```
    /*//////////////////////////////////////////////////////////////
                         TRUSTED REGISTER TESTS
    //////////////////////////////////////////////////////////////*/
```

Reply:
Added better seperator